frontend/src/components/ProjectCardGrid/index.module.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../components/ProjectCard/index.module.scss';
+
+$gap: 16px;
+
+.cardsGrid {
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: $gap;
+  column-gap: $gap;
+  margin: auto;
+  justify-content: center;
+  /* There are three cards per row, so there are to gaps in each row. */
+  max-width: calc($project-card-width * 3 + $gap * 2);
+}
+
+.cardsGrid > div {
+  flex: 0 0 0 33.333%;
```

Comment:
I'll never remember what this means. Could you break this down into individual properties?

Reply:
https://github.com/kitspace/kitspace-v2/pull/504/files#diff-b96cdfa95b55b447ea0c6c06b30aaa71e137d1e4af5c2e39eec5030ade7ba933R17